Testscripts/Linux/utils.sh

```diff
@@ -2467,16 +2467,37 @@ function install_iperf3 () {
 			# For SUSE, keep iperf and iperf3 in the same way
 			ln -s /usr/bin/iperf3 /usr/bin/iperf
 			# iperf3 is not available in the repository of SLES 12
-			which iperf3
+			command -v iperf3
 			if [ $? -ne 0 ]; then
-				LogMsg "iperf3 is not installed. So, Installing iperf3 using rpm"
-				iperf_url="$PACKAGE_BLOB_LOCATION/iperf-sles-x86_64.rpm"
-				libiperf_url="$PACKAGE_BLOB_LOCATION/libiperf0-sles-x86_64.rpm"
-				rpm -ivh $iperf_url $libiperf_url
-				which iperf3
+				iperf3_version=3.2
+				iperf3_url=https://github.com/esnet/iperf/archive/$iperf3_version.tar.gz
+				update_repos
+				gcc -v
```

Comment:
use `command` to check for utility. It can be translated to single line. Similar code exists in this PR
`command -v <utility> > /dev/null ||  install_package <package> `